### ISSUE TITLE
Multiple columns in training data

### ISSUE THREAD

**Issue by disdaining** [kratzert/finetune_alexnet_with_tensorflow#3]
Can you suggest a code snippet for multiple columns in the training file. Example train.txt contains one int column, what if i have more than one floating columns?

Thanks

**Comment by kratzert** [comment]
I'm not sure if I understand the point of your question in this context. For finetuning AlexNet you don't need anything more than an image path and a class label. If it's about Python and reading text files in general, I'm not sure if this is the right place for answering such questions.

Anyway, if you look closely how I implemented reading in data from text files  `datagenerator.py` class I think there is no magic in adding more columns or changing from int to float. I read the text file line-wise and then iterate over the lines, splitting the columns by the separator (in this case a blank space). What you get in return is a list of all elements in this line. If you have 3 columns, well them your  `items` would not only have 2 data points stored (image path and label) but another one. And reading in float numbers, well change the `int()` to `float()`...
